js/data/bucket/circle_bucket.js

```diff
@@ -44,7 +44,8 @@ CircleBucket.prototype.programInterfaces = {
                 return util.premultiply(layer.getPaintValue("circle-color", globalProperties, featureProperties));
             },
             multiplier: 255,
-            paintProperty: 'circle-color'
+            paintProperty: 'circle-color',
+            precision: 'lowp'
```

Comment:
Can the precision qualifiers (lowp, mediump, highp) definitions be moved into the shaders? Maybe something like `#pragma mapbox: define(color, lowp)`? They are only relevant in the context of glsl code.